src/themes/dark.ts

```diff
@@ -33,7 +34,7 @@ export default {
     },
 
     horizontal: {
-      backgroundColor: 'rgba(255, 255, 255, 0.10)',
+      backgroundColor: colors.darkGray,
```

Comment:
The reason the horizontal navbar is transparent is so that it can apply a blur filter for some snaziness. Naming this "darkGrey" doesn't really represent this concept.